src/dsp_tools/models/xmlresource.py

```diff
@@ -11,6 +12,18 @@
 from dsp_tools.models.xmlproperty import XMLProperty
 
 
+@dataclass(frozen=True)
+class BitstreamInfo:
+    """
+    Represents a bitstream object, consisting of it's file name on the local file system,
+    the internal file name assigned by SIPi and optionally its permissions.
```

Comment:
Additionally, I always promote semantic line breaks :-)

Reply:
Done. In this case I'm with you; and in documentation markdown I'm all for semantic line breaks. In docstrings I'm sometimes torn, because I also like docstrings to take up as little screen real estate as possible - and semantic line breaks can easily double or tripple the number of lines the docstring uses up.